src/vsr.zig

```diff
@@ -1381,6 +1381,7 @@ pub const Headers = struct {
         if (std.meta.eql(header.*, Headers.dvc_blank(header.op))) return .blank;
 
         assert(header.command == .prepare);
+        assert(header.invalid() == null);
         if (constants.verify) assert(header.valid_checksum());
```

Comment:
Tests are failing with `operation == .reserved` :joy: 

Also, unrelated to this change, but it probably makes sense to move 

```
        if (constants.verify) assert(header.valid_checksum());
```

up above the other two  assertions.